products/BellHybrid/apps/common/src/layouts/HomeScreenLayoutClassicWithQuotes.cpp

```diff
@@ -124,4 +126,12 @@ namespace gui
         }
     }
 
+    void HomeScreenLayoutClassicWithQuotes::setQuoteText(const UTF8 &quoteContent, const UTF8 &quoteAuthor)
+    {
+        UTF8 authorWithDash = quoteAuthor;
+        authorWithDash.insert(&dash, 0);
```

Comment:
Is `dash` variable used anywhere else? Maybe it can be moved to the local scope of this method?

Reply:
Done